libs/messaging/src/main/kotlin/uk/gov/justice/digital/hmpps/listener/AwsNotificationListener.kt

```diff
@@ -25,7 +31,16 @@ class AwsNotificationListener(
     @WithSpan(kind = SpanKind.CONSUMER)
     fun receive(message: String) {
         try {
-            handler.handle(message)
+            retry(
+                3,
+                listOf(
+                    FeignException.NotFound::class,
+                    RetryableException::class,
+                    ObjectOptimisticLockingFailureException::class,
+                    CannotCreateTransactionException::class,
+                    CannotGetJdbcConnectionException::class
```

Comment:
Is it worth including `CannotAcquireLockException` here too?

Reply:
ok